models/cycle_gan_model.py

```diff
@@ -225,12 +251,33 @@ def compute_D_B_loss(self):
             fake_A = self.fake_A_pool.query(self.fake_A)
             self.loss_D_B = self.compute_D_loss_basic(self.netD_B, self.real_A, fake_A)
 
-
     def compute_D_loss(self):
         """Calculate GAN loss for both discriminators"""
         self.compute_D_A_loss()
         self.compute_D_B_loss()
         self.loss_D = self.loss_D_A + self.loss_D_B
+
+    def compute_D_A_2_loss(self):
```

Comment:
Let's refer to the second discriminator as the 'global' discriminator, e.g. `D_A_global` and `D_B_global` and `D_global` as the option ?

Reply:
We could, but with my implementation `D_A` or `D_A_2` have the same role, the first one can be the global discriminator for example and the second one the patch discriminator.

Reply:
I believe `_global` is less confusing.